samples/XAMLCatalog/XAMLCatalog/UIKitControls/UILabelViewController.h

```diff
@@ -17,5 +17,30 @@
 
 #import "MenuTableViewController.h"
 
-@interface UILabelViewController : MenuTableViewController
+@interface UILabelViewController : UIViewController <UITextFieldDelegate>
+@property (nonatomic) UITextField* text;
+@property (nonatomic) UISlider* fontSize;
+@property (nonatomic) UITextField* x;
+@property (nonatomic) UITextField* y;
+@property (nonatomic) UITextField* w;
+@property (nonatomic) UITextField* h;
```

Comment:
let me know if my explanation in the other comment didn't cover this.

---
In reply to: [104796150](https://github.com/Microsoft/WinObjC/pull/2178#discussion_r104796150) [](ancestors = 104796150)